### PR TITLE
chore(JOML): align with MTE migration

### PR DESCRIPTION
**Pull request by jdrueckert**
https://github.com/Terasology/ModuleTestingEnvironment/pull/34

### CODE REVIEW

File: src/test/java/org/terasology/logic/health/BlockTest.java [hunk 1]
```diff
@@ -33,7 +33,7 @@
 import static org.junit.Assert.assertTrue;
 
 public class BlockTest extends ModuleTestingEnvironment {
-    private static final Vector3i BLOCK_LOCATION = Vector3i.zero().add(Vector3i.down());
+    private static final Vector3i BLOCK_LOCATION = new Vector3i().add(0, -1, 0);
```
Comment (skaldarnar) [inline src/test/java/org/terasology/logic/health/BlockTest.java]: As @keturn pointed out before, the `Vector3i.zero()` was explicit in what the vector is, whereas `new Vector3i()` only implicitly holds that information, and might not be clear from looking at the test case. 

```suggestion
    private static final Vector3ic BLOCK_LOCATION = new Vector3i(0, 0, 0).add(0, -1, 0);
```

**Review by skaldarnar — CHANGES_REQUESTED** [review]

File: src/test/java/org/terasology/logic/health/BlockTest.java (hunk 1 above)
Comment (pollend) [reply src/test/java/org/terasology/logic/health/BlockTest.java]: I also suggest using Vector3ic.

Comment (skaldarnar) [reply src/test/java/org/terasology/logic/health/BlockTest.java]: incorporated Michael's comment in the suggestion 😉

**Review by skaldarnar — DISMISSED** [review]

**Review by skaldarnar — APPROVED** [review]

**Comment by jdrueckert** [comment]
Failing tests were failing before.

**Pull request merged**